yellowpages/yellowpages-war/src/main/webapp/yellowpages/jsp/topicManager.jsp

```diff
@@ -279,7 +280,7 @@ function toAddOrUpdateFolder(action, id) {
 </form>
 <form name="refreshList" action="topicManager"></form>
 
-<div id="folderDialog" style="display:none" title="">
+<div id="folderDialog" style="display:none" title="hgj hjg ">
```

Comment:
Indeed, it is not normal.

Nicolas EYSSERIC
Directeur produit
06 59 55 18 25
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
- - - - - - - - - - - - - - - - -
<http://www.silverpeas.com/>1 place Firmin Gautier
38000 Grenoble
Tél : 04 76 09 31 61
<http://www.facebook.com/pages/Silverpeas/113429255383677>
<http://twitter.com/Silverpeas>- - - - - - - - - - - - - - - - - - - - - -
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


2015-12-14 17:07 GMT+01:00 SilverYoCha <notifications@github.com>:

> In
> yellowpages/yellowpages-war/src/main/webapp/yellowpages/jsp/topicManager.jsp
> <https://github.com/Silverpeas/Silverpeas-Components/pull/413#discussion_r47516136>
> :
>
> > @@ -279,7 +280,7 @@ function toAddOrUpdateFolder(action, id) {
> >  </form>
> >  <form name="refreshList" action="topicManager"></form>
> >
> > -<div id="folderDialog" style="display:none" title="">
> > +<div id="folderDialog" style="display:none" title="hgj hjg ">
>
> Is that normal ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Silverpeas/Silverpeas-Components/pull/413/files#r47516136>
> .
>